pkg/crypto/tinkcrypto/testdata/alice_epk_ref.json

```diff
@@ -0,0 +1,5 @@
+{"kty":"EC",
+  "crv":"P-256",
+  "x":"gI0GAILBdu7T53akrFmMyGcsF3n5dO7MmwNBHKW5SV0",
+  "y":"SLW_xSffzlPWrHEVI30DHM_4egVwt3NQqeUD7nMFpps",
+  "d":"0_NxaRPUMQoAJt50Gz8YiTr8gRTwyEaCumd-MToTmIo"}
```

Comment:
should format these JSON files (e.g., } on new line).

Reply:
will do

Reply:
updated